src/Headers/GhostNeighbours.hpp

```diff
@@ -324,7 +325,7 @@ class GhostNeighbourFinder
 	{
 		if (_any_special){
 		  for (int k=0; k < ndim; k++){
-		    FLOAT dr_k = p.r[k] - cell.rcell[k] ;
+		    FLOAT dr_k = p.r[k] - 0.5*(cell.bb.min[k] + cell.bb.max[k]);  //cell.rcell[k] ;
```

Comment:
Actually it is a function (in TreeCell but can't remember the name).  Maybe I changed this one before making the function and then forgot to go back and correct it, so it just slipped through the net.